packages/flame_forge2d/lib/forge2d_game.dart

```diff
@@ -8,6 +8,7 @@ class Forge2DGame extends FlameGame {
     double zoom = defaultZoom,
     Camera? camera,
     ContactListener? contactListener,
+    super.pauseWhenBackgrounded,
```

Comment:
Do we need to add this to other subclasses?

Reply:
I'd say don't have in the constructor at all and just default the boolean directly in the variable